### ISSUE TITLE
Unable to build latest Ubuntu 18.04 image - "Could not open input file: composer-setup.php"

### ISSUE THREAD

**Issue by ropnop** [actions/virtual-environments#146]
**Describe the bug**
I am trying to use Packer to build the latest Ubuntu 1804 image from `/images/linux/ubuntu1804.json`, but the installation is failing and Packer is aborting with this error:

```
azure-arm: The following NEW packages will be installed:
    azure-arm:   snmp
    azure-arm: 0 upgraded, 1 newly installed, 0 to remove and 30 not upgraded.
    azure-arm: Need to get 0 B/158 kB of archives.
    azure-arm: After this operation, 563 kB of additional disk space will be used.
    azure-arm: Selecting previously unselected package snmp.
    azure-arm: (Reading database ... 241772 files and directories currently installed.)
    azure-arm: Preparing to unpack .../snmp_5.7.3+dfsg-1.8ubuntu3.3_amd64.deb ...
    azure-arm: Unpacking snmp (5.7.3+dfsg-1.8ubuntu3.3) ...
    azure-arm: Setting up snmp (5.7.3+dfsg-1.8ubuntu3.3) ...
    azure-arm: Processing triggers for man-db (2.8.3-2ubuntu0.1) ...
    azure-arm: Installer corrupt
    azure-arm: Could not open input file: composer-setup.php
```

**Virtual environments affected**
- [ ] macOS 10.15
- [ ] Ubuntu 16.04 LTS
- [x ] Ubuntu 18.04 LTS
- [ ] Windows Server 2016 R2
- [ ] Windows Server 2019

**Expected behavior**
A successful `packer build` to create the Ubuntu 1804 image

**Actual behavior**
An installation step fails and Packer aborts the build with this error:
```
azure-arm: The following NEW packages will be installed:
    azure-arm:   snmp
    azure-arm: 0 upgraded, 1 newly installed, 0 to remove and 30 not upgraded.
    azure-arm: Need to get 0 B/158 kB of archives.
    azure-arm: After this operation, 563 kB of additional disk space will be used.
    azure-arm: Selecting previously unselected package snmp.
    azure-arm: (Reading database ... 241772 files and directories currently installed.)
    azure-arm: Preparing to unpack .../snmp_5.7.3+dfsg-1.8ubuntu3.3_amd64.deb ...
    azure-arm: Unpacking snmp (5.7.3+dfsg-1.8ubuntu3.3) ...
    azure-arm: Setting up snmp (5.7.3+dfsg-1.8ubuntu3.3) ...
    azure-arm: Processing triggers for man-db (2.8.3-2ubuntu0.1) ...
    azure-arm: Installer corrupt
    azure-arm: Could not open input file: composer-setup.php
```

1. `git pull master`
2. `cd images/linux`
2. `packer build --var-file=packer_vars.json ubuntu1804.json`

Any thoughts or suggestions?

**Comment by majorvin** [comment]
this applies to ubuntu 1604 as well.

**Comment by cubimon** [comment]
Where do you get `packer_vars.json` from/what does it contain? Probably the variables that are used in ubuntu1804.json. I want to build this image to run azure devops tests locally.

**Comment by majorvin** [comment]
@cubimon you can build it  on your own based on the parameters your packer is expecting. e.g. password

**Comment by kaylangan** [comment]
I believe this was addressed in #194.
